libs/login/WordPressLoginFlow/src/main/java/org/wordpress/android/login/widgets/WPLoginInputRow.java

```diff
@@ -79,22 +66,8 @@ private void init(Context context, AttributeSet attrs) {
                 if (a.hasValue(R.styleable.wpLoginInputRow_wpIconDrawable)) {
                     int iconResId = a.getResourceId(R.styleable.wpLoginInputRow_wpIconDrawable,
                             R.drawable.ic_user_grey_24dp);
-                    int tintResId = a.getResourceId(R.styleable.wpLoginInputRow_wpIconDrawableTint,
-                            R.color.login_input_icon_color);
-
-                    if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.LOLLIPOP) {
-                        mIcon.setImageTintList(ColorStateList.valueOf(ContextCompat.getColor(context, tintResId)));
-                        mIcon.setImageResource(iconResId);
-                    } else {
-                        Drawable drawable = context.getResources().getDrawable(iconResId);
-                        DrawableCompat.setTint(drawable, context.getResources().getColor(tintResId));
-                        DrawableCompat.setTintMode(drawable, PorterDuff.Mode.SRC_IN);
-                        mIcon.setImageDrawable(drawable);
-                    }
 
-                    mIcon.setVisibility(View.VISIBLE);
-                } else {
-                    mIcon.setVisibility(View.GONE);
+                    mTextInputLayout.setStartIconDrawable(iconResId);
```

Comment:
Out of curiosity, I see you "activated" the start icon here. Also I see we removed the `wpIconDrawable` so I was wondering if it was considered to use the start icon instead of totally removing them like below:

![image](https://user-images.githubusercontent.com/47797566/74614568-1f564080-5119-11ea-82af-5d989e4b6aa0.png)

Reply:
Good catch! Yes, we decided to remove the icons after those changes, so it's safe to remove 👍